docs/eventing/channels/channel-types-defaults.md

```diff
@@ -36,10 +58,8 @@ data:
           replicationFactor: 1
 ```
 
-Default channels can be configured for the cluster, a namespace on the cluster, or both.
-
-**NOTE:** If a default channel implementation is configured for a namespace, this will overwrite the configuration for the cluster.
+**NOTE:** `InMemoryChannel` channels should not be used in production environments.
```

Comment:
```suggestion
**NOTE:** `InMemoryChannel` channels must not be used in production environments.
```

Reply:
I'll clarify with "`InMemoryChannel` channels are not suited for production environments."

Reply:
I'd keep it more prescriptive by using something definite like "must not" tbh

Reply:
It might help to explain why not here, e.g. no delivery reattempts, that kinda thing